packages/manager/modules/dbaas-logs/src/logs/detail/aliases/home/logs-aliases-home.controller.js

```diff
@@ -22,10 +23,21 @@ export default class LogsAliasesHomeCtrl {
     this.serviceName = this.$stateParams.serviceName;
     this.LogsAliasesService = LogsAliasesService;
     this.LogsIndexService = LogsIndexService;
+    this.LogsDetailService = LogsDetailService;
     this.CucControllerHelper = CucControllerHelper;
     this.CucCloudMessage = CucCloudMessage;
   }
 
+  $onInit() {
+    this.LogsDetailService.getServiceDetails(this.serviceName).then(
+      (service) => {
+        this.columnsParameters = [
+          { name: 'isEditable', hidden: service.isIamEnabled },
+        ];
+      },
+    );
+  }
+
```

Comment:
This call is already done in `dbaas-logs.detail` routing definition. No need to call it one more time, just bind `service` defined in the resolve section in your component

```suggestion
  $onInit() {
    this.columnsParameters = [
          { name: 'isEditable', hidden: this.service.isIamEnabled },
        ];
  }

```

Reply:
It's probably because you've forgotten to bind the property into your component

current
```
export default {
  controller,
  controllerAs: 'ctrl',
  template,
};
```
expected
```
export default {
  bindings: {
    service: '<',
  }, 
  controller,
  controllerAs: 'ctrl',
  template,
};
```